Resources/public/js/views/services/ez-contentcreateviewservice.js

```diff
@@ -22,6 +22,10 @@ YUI.add('ez-contentcreateviewservice', function (Y) {
      * @extends eZ.ContentEditViewService
      */
     Y.eZ.ContentCreateViewService = Y.Base.create('contentCreateViewService', Y.eZ.ContentEditViewService, [], {
+        initializer: function () {
+            this.on('*:changeLanguage', this._changeLanguage);
```

Comment:
bad method naming, the method should be named `selectLanguage` so that you directly know what happen when the `changeLanguage` event is fired.